docs/root/configuration/http/http_filters/_include/aws_credentials.rst

```diff
@@ -18,10 +18,17 @@ secret access key (the session token is optional).
    ``Expiration``). To enable this credentials provider set ``envoy.reloadable_features.use_http_client_to_fetch_aws_credentials`` to ``true``
    so that it can use http async client to fetch the credentials. This provider is not compatible with :ref:`Grpc Credentials AWS AwsIamConfig
    <envoy_v3_api_file_envoy/config/grpc_credential/v3/aws_iam.proto>` plugin which can only support deprecated libcurl credentials fetcher
-   , see https://github.com/envoyproxy/envoy/pull/30626. To fetch the credentials a static cluster is required with the name
-   ``sts_token_service_internal`` pointing towards regional AWS Security Token Service. The static internal cluster will still be added even
-   if initially ``envoy.reloadable_features.use_http_client_to_fetch_aws_credentials`` is not set so that subsequently if the reloadable feature
-   is set to ``true`` the cluster config is available to fetch the credentials.
+   , see https://github.com/envoyproxy/envoy/pull/30626. To fetch the credentials a static cluster is created with the name
```

Comment:
nit: not realted to your change, but can you please move the comma to the line above (or the word "fetcher" to this line)?